api/v1beta1/azurecluster_validation_test.go

```diff
@@ -1024,9 +1024,11 @@ func TestValidateNodeOutboundLB(t *testing.T) {
 			name: "FrontendIps can update when frontendIpsCount changes",
 			lb: &LoadBalancerSpec{
 				FrontendIPs: []FrontendIP{{
-					Name: "some-frontend-ip-1",
+					Name:             "some-frontend-ip-1",
+					PrivateIPAddress: "1.2.3.4",
```

Comment:
is this test related to the changes?

Reply:
no will remove it, was added in previous changes